doc/user/content/release-notes.md

```diff
@@ -110,6 +110,8 @@ changes that have not yet been documented.
 - Fix parsing of nested empty `SELECT` statements, as in
   `SELECT * FROM (SELECT)` {{% gh 8723 %}}.
 
+- Fix parsing of repeat constant expressions in a `GROUP BY` clause {{% gh 8302 %}}
```

Comment:
Materialize pedantry: this was a planning bug, not a parsing bug!

Reply:
pedantry is greatly appreciated! thanks for pointing this one out